pruntime/enclave/src/contracts/balance.rs

```diff
@@ -113,6 +172,22 @@ impl contracts::Contract<Command, Request, Response> for Balance {
                     }
                     Ok(Response::FreeBalance { balance })
                 },
+                Request::TransferToChain {sequence} => {
```

Comment:
This query returns the messages in the queue. So better call it something like `MessageQueue` or `PendingChainTransfer`